apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/template-description.tsx

```diff
@@ -62,12 +62,82 @@ export const TemplateDescription: FunctionComponent<TemplateDescriptionPropsInte
     } = props;
 
     const { t } = useTranslation();
+    const { getLink } = useDocumentation();
 
+    const resolveDocumentationLink = (): ReactElement => {
+        const templateName = template?.name;
+
+        if ( templateName === "User-Age-Based" ) {
```

Comment:
Lets move these `"User-Age-Based"` etc. to an enum(`AdaptiveAuthTemplateTypes`). Also better to refactor as follows to avoid duplication. WDYT?

let docLink: string = undefined;

```suggestion
        if ( templateName === AdaptiveAuthTemplateTypes. USER_AGE_BASED) {
            docLink = getLink(<LINK>);
        }
        
        ...
        
        if (!doclink) {
            return null;
        }
        
        return (
            <DocumentationLink
                link={ docLink }
            >
                 { t("common:learnMore") }
            </DocumentationLink>
        );
```

Reply:
Fixed with 249e320604fcef016aaffc5e2bb956c0cd74921f